worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
```

Comment:
Not really.

`GetLength()` indicates a static value in `SdesItem`. It represents the length of the item value, which cannot be changed.

`GetSize()` is dynamic in the sense that a when a RTCP packet is modified, ie: adding an item, removing an item, etc, the size of the element changes.